src/hooks.js

```diff
@@ -0,0 +1,41 @@
+import getLearnerPortalLinks from './learnerPortalLinks';
+import { getSelectedEnterpriseUUID } from './utils';
+
+export default function useEnterpriseConfig(
+  authenticatedUser,
+  getAuthenticatedHttpClient,
+  useEffect,
+  useState,
```

Comment:
I ran into "Invalid hook call" issues trying to install react into this package and then install this package into `frontend-app-learner` https://reactjs.org/docs/error-decoder.html/?invariant=321 ...I think because react-dom had issues determining which react to use (the one in this package, or the one in frontend-app-learning).  Injecting the `useState` and `useEffect` functions from the client of this package fixed those up.

I ran into an error "no property `getAuthenticatedHttpClient()` of undefined" when I had frontend-platform installed, so I took the same injection approach with that, too.